### PR TITLE
Adds unique ammo effects to launcher ammo

### PR DESCRIPTION
**Pull request by Brian-Otten**
#### Summary

```SUMMARY: Content "Adds unique ammo effects to launcher ammo"```

#### Purpose of change
Launcher ammo was using generic ammo effects, and it was causing many of the launchers to feel and behave weirdly. this updates all launcher ammo to have their own new ammo effect with the correct explosive force, casing weight, and estimated fragment size.

#### Describe the solution
This updates the launcher ammo to all have their own new ammo effect with the correct explosive force, casing weight, and estimated fragment size. This has buffed some of the heavier launchers in particular to be a decent bit more explosives, while making the 40mm grenades much more manageable.

Explosively fragmenting launcher ammo with a range of under 60 has been buffed to range 60, so that the effective casualty radius of your rocket is no longer larger than your weapon range.

Most of the data needed for determining the explosive power was able to be reused from #37052

#### Testing
Load up a world, use debug mutations to give myself infinite carry space, give myself all the launchers, and without giving myself invulnerability, try to somewhat clear a town.

With some careful aiming, i managed to make effective use of all the launchers without my character getting hurt.

I also made note of all the explosion sizes to make sure all of them were making sense.

### CODE REVIEW

File: data/json/ammo_effects.json [hunk 1]
```diff
@@ -115,18 +120,88 @@
...
-    }
+    "explosion": { "power": 2.4, "shrapnel": { "casing_mass": 28, "fragment_mass": 14 } }
   },
   {
     "id": "FRAG",
     "type": "ammo_effect",
     "explosion": { "power": 185, "shrapnel": { "casing_mass": 212, "fragment_mass": 0.05 } }
   },
+  {
+    "id": "20x66_FRAG",
```
Comment (ZhilkinSerg) [inline data/json/ammo_effects.json:131]: I would prefer using EXPLOSION/FRAG prefix instead of suffix.

File: data/json/ammo_effects.json [hunk 2]
```diff
@@ -115,18 +120,88 @@
...
+    "type": "ammo_effect",
+    "explosion": { "power": 40, "shrapnel": { "casing_mass": 15, "fragment_mass": 0.15 } }
+  },
+  {
+    "id": "20x66_EXPLOSIVE",
```
Comment (tenmillimaster) [inline data/json/ammo_effects.json:136]: TBH, these should be migrated/obsoleted, not updated.
They're too small to deliver a useful payload, and explosive small arms ammunition is quite firmly a 'war crime' sort of thing, so it's a non-starter for even being a prototype.

File: data/json/ammo_effects.json (hunk 1 above)
Comment (Brian-Otten) [reply data/json/ammo_effects.json]: Changed all the ammo effects to use prefixes

File: data/json/ammo_effects.json (hunk 2 above)
Comment (Brian-Otten) [reply data/json/ammo_effects.json]: I'm not entirely against, but i'm not 100% on rivtechs role and in how much these things are for fighting nether creatures, and if this PR merges they will at least have the proper ammo effect when/if they get adopted by aftershock.

But more importantly, i won't have time to replace their spawns in the loot pools with appropriate alternatives everywhere and i don't want to obsolete it if i'm going to half ass it.

**Pull request merged**